options.html

```diff
@@ -82,6 +82,16 @@ <h3>Shortcuts</h3>
                 <option value="false">Do not disable website key bindings</option>
                 <option value="true">Disable websites key bindings</option>
             </select></div>
+        <div class="row customs" id="reposition">
+            <select class="customDo">
+                <option value="reposition">Reset controller position</option>
+            </select>
+            <input class="customKey" type="text" value="" placeholder="press a key">
+            <input class="customValue" type="text" placeholder="value (0.10)">
```

Comment:
Is that the right placeholder value?

Reply:
I was unsure about what to put down as the placeholder value for this, so I just put in anything for the time being. Not sure if this needs any placeholder at all, though

Reply:
@igrigorik how do you feel about keeping that "dummy" value there? Or should I change it somehow